addons/account/models/account.py

```diff
@@ -451,6 +451,23 @@ def unlink(self):
         partner_prop_acc = self.env['ir.property'].search([('value_reference', 'in', values)], limit=1)
         if partner_prop_acc:
             raise UserError(_('You cannot remove/deactivate an account which is set on a customer or vendor.'))
+        default_journals = self.env['account.journal'].search([
```

Comment:
use ondelete restrict

Reply:
@pimodoo Is there a way to keep the error message like this with the ondelete="restrict" parameter?